packages/devextreme/js/ui/context_menu/ui.context_menu.js

```diff
@@ -643,6 +644,7 @@ class ContextMenu extends MenuBase {
 
     _initScrollable($container) {
         this._createComponent($container, Scrollable, {
+            useKeyboard: false,
```

Comment:
What about tests for this?
It also should declare that Axe reports an error is scrollable has tabindex